blueocean-web/src/main/java/io/jenkins/blueocean/BlueOceanUI.java

```diff
@@ -56,13 +58,6 @@ public String getUrlBase() {
     }
 
     /**
-     * Have some slightly different behavior in development mode
-     */
-    public boolean isDevelopmentMode() {
```

Comment:
:bug: this is [referenced in `index.jelly`](https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-web/src/main/resources/io/jenkins/blueocean/BlueOceanUI/index.jelly#L63) to provide this info to the front end